docs/how-to-guides/jdbc-cosmos-notes.md

```diff
@@ -62,6 +62,20 @@ client.get_offline_features(...)
 
 These values will be automatically passed to the Feathr core when submitting the job.
 
+If you want to use token, the code will be like this:
+```
+src1_name="source1"
+source1 = JdbcSource(name=src1_name, url="jdbc:...", dbtable="table1", auth="TOKEN")
+anchor1 = FeatureAnchor(name="anchor_name",
```

Comment:
Could the 1 postfix in src1, source1, anchor1 get removed?

Reply:
This part aligns with the `USERPASS` sample. I think the purpose of 1 postfix is to indicate that there could be multiple sources. let me update it with 2 then.